src/js/module/Clipboard.js

```diff
@@ -14,59 +39,50 @@ define([
      * @param {Event} event
      */
     var hPasteClipboardImage = function (event) {
+
       var clipboardData = event.originalEvent.clipboardData;
       var layoutInfo = dom.makeLayoutInfo(event.currentTarget || event.target);
       var $editable = layoutInfo.editable();
 
       if (!clipboardData || !clipboardData.items || !clipboardData.items.length) {
+
         var callbacks = $editable.data('callbacks');
         // only can run if it has onImageUpload method
         if (!callbacks.onImageUpload) {
           return;
         }
 
-        // save cursor
-        handler.invoke('editor.saveNode', $editable);
-        handler.invoke('editor.saveRange', $editable);
-
-        $editable.html('');
-
         setTimeout(function () {
-          var $img = $editable.find('img');
-
-          // if img is no in clipboard, insert text or dom
-          if (!$img.length || $img[0].src.indexOf('data:') === -1) {
-            var html = $editable.html();
-
-            handler.invoke('editor.restoreNode', $editable);
-            handler.invoke('editor.restoreRange', $editable);
-
-            handler.invoke('editor.focus', $editable);
-            try {
-              handler.invoke('editor.pasteHTML', $editable, html);
-            } catch (ex) {
-              handler.invoke('editor.insertText', $editable, html);
-            }
+          if (!pasteDiv) {
+            return;
+          }
+          
+          var $img = pasteDiv[0].childNodes[0];
```

Comment:
This time `$img` is not jQuery element, It's better to use `imgNode` instead.
`var imgNode = $pasteDiv[0].firstChild;`